.github/workflows/label-prs.yml

```diff
@@ -0,0 +1,95 @@
+name: Label PR
+
+on:
+  pull_request:
+    types: [assigned, opened, edited, synchronize, reopened]
+
+jobs:
+  validate-branch-name:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate branch name
+        env:
+          HEAD_REF: ${{ github.event.pull_request.head.ref }}
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const BRANCH_REGEX = /^main|develop|(ci|chore|docs|feat|fix|perf|refactor|revert|style)\/\d*(?:[-](?![-])\w*)*$/;
```

Comment:
would be nice to have it in readMe or developer guide for creating the branch, wdyt?